beta/aws-ecsfargate-cfn/README.md

```diff
@@ -0,0 +1,220 @@
+
+## 🏗️ Deploy 1Password SCIM Bridge
+
+Use the AWS CloudFormation console to [create a stack](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stack.html) or deploy 1Password SCIM Bridge [using AWS CLI](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-cli.html):
+
+### 🖼 Use the AWS CloudFormation console
+
+For a GUI deployment, you can create a stack in the CloudFormation console:
+
+1. [Create a new stack](https://console.aws.amazon.com/cloudformation/home#/stacks/create) in the CloudFormation console.
+2. In the "Specify template" section, choose "Upload a template file".
+3. Click Choose File.
+4. Upload the `op-scim-bridge.yaml` file from the working directory on your computer.
+5. Click Next.
+6. Type a name for the stack in the "Stack name" field.
+   > **Note**
+   >
+   > 📄 We use `op-scim-bridge` as a shorthand for 1Password SCIM Bridge throughout our examples and supporting documentation, but the choice is arbitrary. CloudFormation will use the stack name (or a truncated version where needed) as a prefix when naming the created AWS resources.
+7. Leave the default value already supplied for "VPC CIDR range" unless you require something different for your environment.
+8. Select the Route 53 hosted zone in which to create DNS records.
+9. Replace `scim.example.com` with a domain name for your SCIM bridge that is in the domain of this hosted zone.
+10. Open the `scimsession` file in a text editor. Select all text in the file and copy it to your clipboard.
+11. Paste the contents into the `scimession` field (it will be masked on input). Click Next.
+12. Optionally, click "Add new tag" to add tags to all supported resources in the stack. Click Next.
+13. Review the stack configuration. Under Capabilities, check "I acknowledge that AWS CloudFormation might create IAM resource.". Click Submit.
+
+The console will display the stack status as `ℹ️ CREATE_IN_PROGRESS` during the deployment. The stack is expected to take several minutes to create (generally >5 minutes in testing at the time of the last review). After the stack is created, the status will change to `✅ CREATE_COMPLETE`.
+
+### 💻 Use AWS CLI
+
+You can alternatively deploy 1Password SCIM Bridge with a single AWS CLI command.
+
+*Example command:*
+
+```sh
+aws cloudformation deploy \
+    --template-file ./op-scim-bridge.yaml \
+    --stack-name op-scim-bridge \
+    --capabilities CAPABILITY_IAM \
+    --parameter-overrides \
+        Route53HostedZoneID=Z2ABCDEF123456 \
+        DomainName=scim.example.com \
+        scimsession=$(cat ./scimsession) \
+        # VPCCIDR=10.0.0.0/16 \
+    # --tags \
+    #     Key1=Value1 \
+    #     Key2=Value2
+```
+
+1. If the template file was saved with a different file name or somewhere other than the working directory, replace `./op-scim-bridge.yaml` with the actual path (for example, `--template-file path/to/template.file`).
```

Comment:
The steps here make it seem like they should use the example as a template. So we could lead in with that:

```suggestion
Edit the example above with the following:

1. If the template file was saved with a different file name or somewhere other than the working directory, replace `./op-scim-bridge.yaml` with the actual path (for example, `--template-file path/to/template.file`).
```